build/syntax-highlight.js

```diff
@@ -80,7 +80,10 @@ function syntaxHighlight($, doc) {
     const code = $pre.text();
     const html = Prism.highlight(code, grammar, name);
     const $code = $("<code>").html(html);
+    const wrap = $("<div class='code-example'></div>");
```

Comment:
I haven't actually checked out the code but I suspect we can merge this with the line above. 
It actually used to be that Prism would produce a blob of HTML and it would be injected straight into the `<pre>`. So you'd end up with:
```html
<pre class="brush: js"><span class="prism-token">function</span> ...
```
But then we decided all the code should be wrapped in a `<code>` too, so it became:
```html
<pre class="brush: js"><code><span class="prism-token">function</span> ...
```

That's what that `$code` variable is above. Perhaps you can combine these two into one `cheerio` operation. And I like the name `wrap` better than `$code` :)

Reply:
That's interesting to see how the Prism usage has evolved over time. I think your right, `$code` and `wrap` can be combined into one operation, I like the name `wrap` better than `$code` too :D 

we can also get rid of the `.wrapAll()` as well like you stated below.